Helper/Order.php

```diff
@@ -762,42 +762,30 @@ public function saveUpdateOrder($reference, $storeId = null, $boltTraceId = null
     }
 
     /**
-     * @param \Magento\Quote\Model\Quote $quote
-     * @param string                     $transaction
-     * @return OrderModel
-     * @throws BoltException
-     * @throws LocalizedException
+     * Try to fetch and price-validate already existing order.
+     * Use case: order has been created but the payment failes due to the ivnalid credit card data.
+     * Then the customer enters the correct card info, the previously created order is used if the amounts match.
+     *
+     * @param Quote $quote
+     * @param \stdClass $transaction
+     * @return bool|false|OrderModel
```

Comment:
As i see we alway return here of `false` or `OrderModel `.
Can leave here only `false|OrderModel`? How do you think?